lib/site-builder.js

```diff
@@ -290,13 +303,18 @@ exports.launchBuilder = function(info, builderConfig, done) {
 };
 
 exports.makeBuilderListener = function(webhook, builderConfig, done) {
-  var org = builderConfig.githubOrg || config.githubOrg;
-
-  webhook.on('refs/heads/' + builderConfig.branch, function(info) {
-    if (info.repository.organization === org) {
-      exports.launchBuilder(info, builderConfig, done);
-    }
-  });
+  var org = builderConfig.githubOrg || config.githubOrg,
```

Comment:
While I'm usually a fan of using comma, separated var declarations, this one looks a little messy when they're functions. I'd suggest this:

```js
var whatever = 'whatever',
    handler;

handler = function() {
...
```